### PR TITLE
268 DISP-S1 stub implementation

### PR DESCRIPTION
**Pull request by david-inglish**
## Description
- This pull request implements the items in the task list for issue 268.
 
## Affected Issues
- #268 

## Testing
- Added .ci/scripts/test_disp_s1.sh unit test

### CODE REVIEW

File: .ci/docker/Dockerfile_disp_s1 [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ export licenses, or other export authority as may be required, before\
+ exporting such information to foreign countries or providing access to\
+ foreign persons." \
+  org.label-schema.name="OPERA Product Generation Executable (PGE) Image" \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.vendor="California Institute of Technology" \
+  maintainer="California Institute of Technology"
+
+# Copy the OPERA PGE software into the container
+# the base container has a default user "compass_user" with UID/GID 1000/1000
```
Comment (collinss-jpl) [inline .ci/docker/Dockerfile_disp_s1:40]: This comment needs to be updated since the user/group is different

File: .ci/scripts/test_disp_s1.sh [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        /workspace/${TEST_RESULTS_REL_DIR} -type d -exec chmod 775 {} +"
+
+    ${DOCKER_RUN} bash -c "find \
+        /workspace/${TEST_RESULTS_REL_DIR} -type f -exec chmod 664 {} +"
+}
+
+trap set_perms EXIT
+
+# linting and pep8 style check (configured by .flake8 and .pylintrc)
+${DOCKER_RUN} bash -c "source /usr/local/bin/_activate_current_env.sh; flake8 \
```
Comment (collinss-jpl) [inline .ci/scripts/test_disp_s1.sh:64]: Nice work tracking down this activation script to make this work 👍

File: examples/disp_s1_sample_runconfig-v3.0.0-er.1.0.yaml [hunk 3]
```diff
@@ -0,0 +1,756 @@
...
+                IsoTemplatePath:
+
+            QAExecutable:
+                Enabled: False
+                ProgramPath:
+                ProgramOptions: []
+
+            DebugLevelGroup:
+                DebugSwitch: False
+                ExecuteViaShell: True  # Must be set to True for test to work
```
Comment (collinss-jpl) [inline examples/disp_s1_sample_runconfig-v3.0.0-er.1.0.yaml:46]: This should be set to False and the comment removed for the "example" runconfigs

File: src/opera/pge/disp_s1/schema/algor_param_disp_s1_schema.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            # Options for `create_dataset` with h5py.
+            hdf5_creation_options:
+                chunks: list()
+                compression: str()
+                compression_opts: int()
+                shuffle: bool()
+            # GDAL creation options for GeoTIFF files.
+            gtiff_creation_options: list()
+        # Name of the subdataset to use in the input NetCDF files.
+        subdataset: science/SENTINEL1/CSLC/grids/VV
```
Comment (collinss-jpl) [inline src/opera/pge/disp_s1/schema/algor_param_disp_s1_schema.yaml:60]: Think this needs to be set to `str()`?

**Review by collinss-jpl — COMMENTED** [review]
Hi @david-inglish, I assume you're still putting the finishing touches on this branch, but I wanted to give it a quick once-over and found a few minor issues. Everything mostly looks great though!

**Comment by collinss-jpl** [comment]
I just took a look at the CI job for this branch, and the test that is failing in test_usage_metrics.py should be ignored (I commented out the particular assertion line for the last release). I think since we've changed instance types for the CI machine the assumption that was tested no longer holds. I will not let it hold up merging of this branch.

File: .ci/docker/Dockerfile_disp_s1 (hunk 1 above)
Comment (david-inglish) [reply .ci/docker/Dockerfile_disp_s1:40]: Done

File: examples/disp_s1_sample_runconfig-v3.0.0-er.1.0.yaml (hunk 3 above)
Comment (david-inglish) [reply examples/disp_s1_sample_runconfig-v3.0.0-er.1.0.yaml:46]: Done

File: src/opera/pge/disp_s1/schema/algor_param_disp_s1_schema.yaml (hunk 4 above)
Comment (david-inglish) [reply src/opera/pge/disp_s1/schema/algor_param_disp_s1_schema.yaml:60]: Yes, good catch!

**Pull request merged**